config/rootfs/debos/overlays/chromeos-flash/opt/chromeos/install-modules

```diff
@@ -18,7 +18,7 @@ install_modules()
     cd "$mount_dir"
     ls -l lib/modules
     rm -rf lib/modules
-    tar xf "$root_path/$modules_tarball"
+    tar --no-same-owner --no-same-permissions --no-xattrs --no-selinux --no-acls -xf "$root_path/$modules_tarball"
     chown root: -R lib
```

Comment:
Guess we may be able to drop this now. I'm testing it with a debug job in LAVA at the moment.

Reply:
Yes I confirm this can be dropped.  Merging this now anyway to complete the rebase of the `chromeos` branch and we'll then be able to consolidate the whole thing again.